### PR TITLE
Remove Fable.Import.NodeLibzfs

### PR DESCRIPTION
**Pull request by jgrund**
We are no longer using it.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,75 +1,56 @@
 sudo: required
 language: generic
 services:
-  - docker
-addons:
-  apt:
-    update: true
+    - docker
```
Comment (johnsonw) [inline .travis.yml]: I guess the tab changed?

File: libzfs-sys/build.rs [hunk 2]
```diff
@@ -8,6 +8,10 @@ extern crate pkg_config;
 use std::env;
 
 fn main() {
+    if cfg!(target_os = "macos") {
```
Comment (johnsonw) [inline libzfs-sys/build.rs]: Should we whitelist instead of blacklist?

Comment (jgrund) [reply libzfs-sys/build.rs]: maybe, want some more feedback before deciding one way or another.

File: .travis.yml (hunk 1 above)
Comment (jgrund) [reply .travis.yml]: yeah, I'll move this all to two spaces in a future patch.

Comment (jgrund) [reply .travis.yml]: actually, it really wants to be four, which is why that moved.

**Pull request merged**